### DISCUSSION TITLE
How do I reuse the *username* of an acccount *I* perma-deleted a while ago on mastodon.social?

### DISCUSSION THREAD

**Discussion by 103sbavert** [mastodon/mastodon#28757] [Q&A]
Whenever I try to make a new account with said username, it tells me its already taken. But I'm unable to recover it with my email address with the forgot password option.

**Comment by StrangeMinds1960** [comment]
Not sure what you mean by 'perma-deleted', but I deleted a few accounts recently with 'tootctl accounts delete', and I'm finding that the username is retained in the database but the email address used to sign up is not

**Comment by StrangeMinds1960** [comment]
Found this snippet online which might help -

It is possible to re-animate the username using tootctl, e.g.:
tootctl accounts create OLDNAME --email EMAIL --reattach

**Reply by 103sbavert** [reply]
I believe this snippet is meant to be used by the instance admins, in this case mastodon.social?

**Reply by StrangeMinds1960** [reply]
It is, yes

**Comment by ThisIsMissEm** [comment]
For mastodon.social you'll want to email: [staff@mastodon.social](mailto:staff@mastodon.social) requesting account reactivation & be patient.

**Comment by 103sbavert** [comment]
gotcha thanks!
